templates/docs/examples/patterns/navigation/reduced.html

```diff
@@ -0,0 +1,202 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Navigation / Reduced{% endblock %}
+
+{% block standalone_css %}patterns_navigation{% endblock %}
```

Comment:
Seems like something is missing from your standalone SCSS; the spacing appears to be broken 

See [standalone example](https://vanilla-framework-5107.demos.haus/docs/examples/standalone/patterns/navigation/reduced?theme=light#)

![image](https://github.com/canonical/vanilla-framework/assets/46915153/4e5d26dc-0689-4382-8a15-64138bb1f1cb)

Reply:
Yes, we don't need all that specific content on the page. Will clean up the HTML.